lang/en/docs/yarnrc.md

```diff
@@ -0,0 +1,39 @@
+---
+id: docs_configuration_yarnrc
+guide: docs_configuration
+layout: guide
+---
+
+{% include vars.html %}
+
+`.yarnrc` files allow you to configure additional Yarn features. The [`config` command]({{url_base}}/docs/cli/config/) may also be used to set these options. Yarn will merge `.yarnrc` files up the file tree.
+
+### `yarn-offline-mirror` <a class="toc" id="toc-yarn-offline-mirror" href="#toc-yarn-offline-mirror"></a>
+
+```ini
+yarn-offline-mirror ./packages-cache
+```
+
+Maintain offline copies of your packages for more repeatable and reliable builds. [Find more information here]({{url_base}}/docs/offline-mirror/).
+
+Value must be a relative file path, or `false` to disable the mirror (default).
```

Comment:
This option is being added in https://github.com/yarnpkg/yarn/pull/3009. May want to wait for that to be merged, or I can send a PR without this bit first.